flags.go

```diff
@@ -59,18 +66,48 @@ func (b csvFileBackend) Refresh() (map[string]Flag, error) {
 	return readFile(b.filename, "csv", parseFlagsCSV)
 }
 
+// for csv file, update time looks at last modified time on filesystem
+func (b csvFileBackend) UpdateTime() (time.Time, error) {
+	f, err := os.Open(b.filename)
+	if err != nil {
+		return time.Time{}, err
+	}
+	defer f.Close()
+	statinfo, err := f.Stat()
+	if err != nil {
+		return time.Time{}, err
+	}
+	return statinfo.ModTime(), nil
+}
+
+// for json file, update time comes from json directly
+func (b jsonFileBackend) UpdateTime() (time.Time, error) {
```

Comment:
`UpdateTime` should be linked to the time the file was read. Can we maintain this in an member of the `jsonFileBackend` instance?

Reply:
Done..."age" is now returned in the Refresh() method, at same time the file was read